modules/nf-core/krakenuniq/preloadedkrakenuniq/main.nf

```diff
@@ -11,17 +11,16 @@ process KRAKENUNIQ_PRELOADEDKRAKENUNIQ {
     tuple val(meta), path(fastqs)
     path  db
     val ram_chunk_size
-    val save_output_fastqs
+    val save_output_fasta
```

Comment:
I wounder if coudl be called `save_output_reads` to make it more generic?

Reply:
Of course